edx_exams/apps/core/admin.py

```diff
@@ -30,17 +30,20 @@ class ProctoringProviderAdmin(admin.ModelAdmin):
 
 class ExamAdmin(admin.ModelAdmin):
     """ Admin configuration for the Exam model """
-    list_display = ('resource_id', 'course_id', 'provider', 'content_id', 'exam_name',
-                    'exam_type', 'time_limit_mins', 'due_date', 'hide_after_due', 'is_active')
+    list_display = ('course_id', 'provider', 'exam_name', 'exam_type', 'due_date', 'is_active')
+    readonly_fields = ('resource_id', 'course_id')
+    list_filter = ('is_active',)
     search_fields = ('resource_id', 'course_id', 'provider__name', 'content_id', 'exam_name',
                      'exam_type', 'time_limit_mins', 'due_date', 'hide_after_due', 'is_active')
-    ordering = ('course_id', 'exam_name',)
+    ordering = ('-is_active', 'course_id', 'exam_name',)
 
 
 class ExamAttemptAdmin(admin.ModelAdmin):
     """ Admin configuration for the Exam Attempt model """
     list_display = ('user', 'exam', 'attempt_number', 'status', 'start_time',
                     'allowed_time_limit_mins')
+    list_filter = ('status',)
+    readonly_fields = ('user', 'exam')
```

Comment:
should `attempt_number` also be read only?

Reply:
this is the id according the the LTI tool so I figured I'd leave this editable for testing's sake you can't really break anything by changing it right now. We might need to lock this down at some point tho.